quakemigrate/plot/event.py

```diff
@@ -170,10 +172,9 @@ def _plot_waveform_gather(ax, lut, event, idx):
             if not bool(data):
                 continue
             data = data[0].data
-            data[mint_i:]
 
             # Get station specific range for norm factor
-            stat_maxt = event.otime + ttsf[i]*1.5
+            stat_maxt = event.otime + traveltimes[-1][i]*1.5
```

Comment:
Ditto previous comment, though solution here would be different..

Reply:
Something like `max(traveltimes[:, i])`

Reply:
Addressed.